redis/src/sentinel.rs

```diff
@@ -714,6 +731,27 @@ impl LockedSentinelClient {
     }
 }
 
+#[cfg(feature = "tls-rustls")]
+fn patch_with_tls_certificates(
+    mut params: Vec<ConnectionInfo>,
+    certs: TlsCertificates,
+) -> RedisResult<Vec<ConnectionInfo>> {
+    let new_tls_params = retrieve_tls_certificates(certs)?;
+
+    for node in params.iter_mut() {
+        if let crate::ConnectionAddr::TcpTls {
+            host: _,
+            port: _,
+            insecure: _,
+            ref mut tls_params,
+        } = node.addr
+        {
+            *tls_params = Some(new_tls_params.clone());
```

Comment:
what if `tls_params` is already set with some value? this should probably return an error.